### ISSUE TITLE
DCA_max_trading_pairs has no effect

### ISSUE THREAD

**Issue by bluefuzz** [taniman/profit-trailer#1997]
## ProfitTrailer info
Version: 2.1.1

 
Exchange:
- [x ] Bittrex
- [x ] Binance
- [ ] Poloniex

 
Market:
- [x ] BTC
- [ ] ETH
- [ ] USDT
- [ ] BNB
- [ ] XMR

 
Operating System:
- [ ] Windows (please specify version)
- [x ] Ubuntu (16.04 Xenial)
- [ ] Other linux distro (please specify)
- [ ] Mac OS

## Expected Behavior
According to the wiki for `DCA_max_trading_pairs` 'If you lower this limit below the number of coins you already have in DCA, ProfitTrailer will randomly select the ones that stay in DCA.'

I lowered my DCA_max_trading_pairs from 4 to 2 but I still have 3 pairs in DCA (as I did before). I expected one of the pairs back to PAIRS? 

## Current Behavior
Still 3 pairs in DCA.

## Troubleshooting
Restarted bots. No change.

Here's a zip with profittrailer.json, logfile and current settings

[bug.zip](https://github.com/taniman/profit-trailer/files/2259445/bug.zip)

**Comment by bluefuzz** [comment]
Am I right in thinking that 'MAX DCA PAIRS' will only show in the buy column if no other reasons for not buying are not also true such as MIN BUY VOLUME or MAX BUY TIMES? If so that's probably why I did not think it was working because all pairs are at MAX BUY TIMES ...

**Comment by cvthub** [comment]
Yes, 'MAX DCA PAIRS' is one of the latest checks which are executed. Means, if MAX BUY TIMES is true, then MAX DCA PAIRS wont be executed and wont be displayed.

**Comment by cvthub** [comment]
We extended the wiki entry and added some more info concerning this topic. 

https://wiki.profittrailer.com/doku.php?id=faq#what_does_this_message_in_the_buy_sell_strategy_column_mean
